packages/dashboard/src/components/assetModelSelection/assetModelSelect.tsx

```diff
@@ -11,16 +11,22 @@ type AssetModelSelectOptions = {
   client: IoTSiteWiseClient;
   assetModelId: string;
   selectedAssetId?: string;
+  hideTitle?: boolean;
 };
-export const AssetModelSelect = ({ client, assetModelId, selectedAssetId }: AssetModelSelectOptions) => {
+export const AssetModelSelect = ({
+  client,
+  assetModelId,
+  selectedAssetId,
+  hideTitle = false,
+}: AssetModelSelectOptions) => {
   const { updateSelectedAsset } = useModelBasedQuery();
 
   const { assetSummaries } = useAssetsForAssetModel({ assetModelId, client, fetchAll: true });
 
   const selectedAsset = assetSummaries.find(({ id }) => id === selectedAssetId);
 
   return (
-    <FormField label='Asset'>
+    <FormField label={`${!hideTitle ? 'Asset' : ''}`}>
```

Comment:
Didn’t get you exactly, but I added hideTitle because in the new design Asset label(title) is not required at the top of the select dropdown. So, I am not disturbing this component by completely removing the title just added a check to this label.

Reply:
Got it, Now added the label tag with htmlFor and passed controlId to the Select component for accessibility.